internal/core/operations/add_rooms/add_rooms_executor.go

```diff
@@ -81,17 +79,14 @@ func (ex *AddRoomsExecutor) Execute(ctx context.Context, op *operation.Operation
 
 	if executionErr := errGroup.Wait(); executionErr != nil {
 		executionLogger.Error("Error creating rooms", zap.Error(executionErr))
-		if errors.Is(executionErr, serviceerrors.ErrGameRoomStatusWaitingTimeout) {
-			return operations.NewErrReadyPingTimeout(executionErr)
-		}
```

Comment:
This line is useless since CreateRoom does not return ping timeout error anymore